ui/values/strings.go

```diff
@@ -223,4 +223,6 @@ const (
 	StrUsdBittrex                  = "usdBittrex"
 	StrNone                        = "none"
 	StrProposal                    = "proposals"
+	StrAdvancedMode                = "advancedMode"
+	StrClearFeilds                 = "clearAllFields"
```

Comment:
Revert this also. Use the strings locally. 
The localized strings file is not to be edited.